cnxdb/migrations/20170911201035_transform_cnxml_to_html.py

```diff
@@ -18,11 +18,12 @@ def _batcher(seq, size):
         yield seq[pos:pos + size]
 
 
-def should_run(cursor, limit='LIMIT 1'):
+def should_run(cursor, baked=True, raw=True, limit='LIMIT 1'):
```

Comment:
Any chance you can add a docstring here? This is slightly more complex with the additional logic paths.

Reply:
I guess, I'm looking for a narrative explanation now that there are these new boolean conditions that change the previous behavior.

Reply:
Yeah I'll add a docstring, it is confusing...